src/main/java/com/ciandt/techgallery/service/impl/UserServiceTGImpl.java

```diff
@@ -342,29 +342,37 @@ public TechGalleryUser getUserFromProvider(final String userLogin)
   @Override
   public List<UserResponse> getUsersAutoComplete(String userLogin)
```

Comment:
@jfmedeiros1820 What do you think about this method be named as 'findUserByPartialLogin'?
Also, I think it would be better if it returns TechGalleryUser. Then, the controller which calls it should transform it to UserResponse.